Detections/SecurityEvent/ExchangeOABVirtualDirectoryAttributeContainingPotentialWebshell.yaml

```diff
@@ -0,0 +1,43 @@
+id: faf1a6ff-53b5-4f92-8c55-4b20e9957594
+name: Exchange OAB Virtual Directory Attribute Containing Potential Webshell
+description: |
+  'This query uses Windows Event ID 5136 in order to detect potential webshell deployment by exploitation of CVE-2021-27065. This query looks for changes to the InternalHostName or ExternalHostName properties of Exchange OAB Virtual Directory objects in AD Directory Services where the new objects contain potential webshell objects. Ref: https://aka.ms/ExchangeVulns'
+severity: High
+requiredDataConnectors:
+  - connectorId: SecurityEvents
+    dataTypes:
+      - SecurityEvents
+queryFrequency: 1h
+queryPeriod: 1h
+triggerOperator: gt
+triggerThreshold: 0
+tactics:
+  - InitialAccess
+relevantTechniques:
+  - T1190
+query: |
+  SecurityEvent
+  // Look for specific Directory Service Changes
+  | where EventID == 5136
+  | extend Data = parse_xml(EventData)
+  | extend ObjectClass = tostring( Data.EventData.Data[10].["#text"])
```

Comment:
We may want to consider  ```EventData has 'msExchOABVirtualDirectory'``` and then expanding more of the EventData fields so we always have the right index location for this and the later parsing.  You could use the method from the [Sysmon All Events parser](https://github.com/Azure/Azure-Sentinel/blob/master/Parsers/Sysmon/Sysmon-AllVersions_Parser.txt) -
```
| extend EventData = parse_xml(EventData).DataItem.EventData.Data
| mv-expand bagexpansion=array EventData
| evaluate bag_unpack(EventData)
| extend Key=tostring(['@Name']), Value=['#text']
```